tests/test_ocs_basic_install.py

```diff
@@ -181,3 +254,28 @@ def load_config_data(data_path):
     """
     with open(data_path, "r") as data_descriptor:
         return yaml.load(data_descriptor, Loader=yaml.FullLoader)
+
+
+def create_rook_resource(template_name, rook_data, cluster_path):
+    """"
+    Create a rook resource after rendering the specified template with
+    the rook data from cluster_conf.
+
+    Args:
+        template_name: name of the ocs-deployment config template.
+        rook_data: rook specific config from cluster_conf
+        cluster_path: path to cluster directory, where files will be written
+    """
+    base_name = template_name.split('.')[0]
+    template_path = os.path.join('ocs-deployment', template_name)
+    template = render_template(
+        template_path,
+        rook_data.get(base_name, {})
+    )
+    cfg_file = os.path.join(cluster_path, template_name)
+    with open(cfg_file, "w") as f:
+        f.write(template)
+    log.info(f"Creating rook resource from {template_name}")
+    # TODO: logging this just for testing purposes, change to run_cmd
+    log.info(f"oc create -f {cfg_file}")
+>>>>>>> 2993ae6... Coady's changes for ocs deployment
```

Comment:
ha this one, please :-)

Reply:
i think this is an outdated version of the file

Reply:
That is correct, Thanks